The prelude (#1329)
powdr-labs/powdr: book/src/asm/modules.md

{{#include ../../../test_data/asm/book/modules.asm}}
```

> Note that a module can't be called `std`, as this name is reserved for an upcoming powdr standard library.

Reviewer: ```suggestion
Note that a module can't be called `std`, as this name is reserved for the powdr standard library.
```

> Note that a module can't be called `std`, as this name is reserved for an upcoming powdr standard library.

Author: I didn't write that :)

> Similar to Rust, any reference that cannot be resolved is lookup up once more in ``std::prelude``.

Reviewer: ```suggestion
Similar to Rust, any reference that cannot be resolved is looked up once more in ``std::prelude``.
```
?

> This module exposes basic types like ``Option``. This means that you can use `Option` anywhere

Reviewer: ```suggestion
This module exposes basic types such as ``Option``. This means that you can use `Option` anywhere
```

> This module exposes basic types like ``Option``. This means that you can use `Option` anywhere

Author: Why?

> This module exposes basic types like ``Option``. This means that you can use `Option` anywhere

Reviewer 2: Are two ticks doing something else than one tick?

> This module exposes basic types like ``Option``. This means that you can use `Option` anywhere

Author: No but in some dialects of markdown they are required.
